tasks/setup.yaml

```diff
@@ -1,6 +1,35 @@
+variables:
+  - name: REGISTRY1_TEST_IMAGE
+    default: "registry1.dso.mil/ironbank/opensource/defenseunicorns/zarf/zarf-agent"
+  - name: REGISTRY
+    default: "${REGISTRY}"
+  - name: REGISTRY_USERNAME
+    default: "${REGISTRY_USERNAME}"
+  - name: REGISTRY_PASSWORD
+    default: "${REGISTRY_PASSWORD}"
+  - name: REGISTRY_RETRY_INTERVAL
+    default: "${REGISTRY_RETRY_INTERVAL}"
```

Comment:
Would love a real default here instead of the env so that we don't have to pass it in.

Reply:
maybe im being dense but why are the defaults being set like this?

Reply:
Opened a separate thread on a similar issue still 👀. TLDR is I don't think we care about calling this directly since these are common-tasks and should always be included elsewhere.